sdk/python/tests/integration/online_store/test_universal_online.py

```diff
@@ -785,3 +785,26 @@ def assert_feature_service_entity_mapping_correctness(
                 entity_rows=entity_rows,
                 full_feature_names=full_feature_names,
             )
+
+
+@pytest.mark.integration
+@pytest.mark.universal_online_stores(only=["postgres"])
+def test_retrieve_online_documents(
```

Comment:
Will you be outputting the cosine similarity as well? That would be useful possibly for debugging. Would be good to be able to test that the engine computes it...maybe not doable though.

Reply:
Should be possible. Somehow just the integration test doesn't startup the Postgres container. And I'm debugging it.